api/user/controller/update.js

```diff
@@ -0,0 +1,22 @@
+import Joi from 'joi';
+import updateUser from '../service/update.js';
+
+export default function update(req, res) {
+  const userSchema=Joi.object({
```

Comment:
move userSchema constant outside update() function 
in such case function update() became more readable